app/helpers/routerHelper.js

```diff
@@ -6,13 +6,15 @@ export const withCatch = (authProvider, route) => async (req, res, next) => {
   try {
     return await route(req, res, next);
   } catch (err) {
+    if (err instanceof ErrorContext) {
```

Comment:
what is the significance of the order of these `if`s in this file?

Reply:
Just thought we should handle this one first.

Reply:
fair 😄